database/seeds/UserTableSeeder.php

```diff
@@ -50,10 +50,21 @@ public function run()
         // Add Contestant Users
         $waitingRooms = WaitingRoom::all();
         $totalRooms = count($waitingRooms);
-        $seedContestants = $this->northstar->getSeedUsers(300);
+        // $seedContestants = $this->northstar->getSeedUsers(300);
```

Comment:
Actually, we should probs just update that method in `Northstar.php` to be `getUsers($limit = '100', $page = 1)` or whatevs so we have that as a method in there not just for seeding.